workflows/phageannotator/main.nf

```diff
@@ -295,42 +306,49 @@ workflow PHAGEANNOTATOR {
     ch_anicluster_reps_fasta = GUNZIP_CLUSTER_REPS ( ch_anicluster_reps_fasta_gz ).gunzip
     ch_versions = ch_versions.mix( GUNZIP_CLUSTER_REPS.out.versions )
 
-    if ( !params.skip_iphop ){
+    // if run_iphop == true, run subworkflow
+    if ( params.run_iphop ){
         // create channel from params.checkv_db
         if ( !params.iphop_db ){
             ch_iphop_db = null
         } else {
             ch_iphop_db = file( params.iphop_db, checkIfExists:true )
         }
 
-    //
-    // SUBWORKFLOW: Download database and predict phage hosts
-    //
-    ch_host_predictions_tsv = FASTA_PHAGE_HOST_IPHOP ( ch_anicluster_reps_fasta, ch_iphop_db ).host_predictions_tsv
-    ch_versions = ch_versions.mix( FASTA_PHAGE_HOST_IPHOP.out.versions )
+        //
+        // SUBWORKFLOW: Download database and predict phage hosts
+        //
+        ch_host_predictions_tsv = FASTA_PHAGE_HOST_IPHOP ( ch_anicluster_reps_fasta, ch_iphop_db ).host_predictions_tsv
+        ch_versions = ch_versions.mix( FASTA_PHAGE_HOST_IPHOP.out.versions )
     } else {
+        // if run_iphop == false, skip subworkflow
         ch_host_predictions_tsv = Channel.empty()
     }
 
 
     /*----------------------------------------------------------------------------
         Predict virus lifestyle
     ------------------------------------------------------------------------------*/
-    if ( !params.skip_bacphlip ) {
+    // if run_bacphlip == true, run subworkflow
+    if ( params.run_bacphlip ) {
         //
         // MODULE: Predict phage lifestyle using lysogeny specific genes
         //
         ch_bacphlip_lifestyle_tsv = BACPHLIP ( ch_anicluster_reps_fasta ).bacphlip_results
         ch_versions = ch_versions.mix( BACPHLIP.out.versions )
+
+        // TODO: Add ability to automatically add -meta flag to pharokka when multiple sequences are in input fasta
```

Comment:
Addressed below? Does BACPHLIP also run prokka?

Reply:
Not addressed yet. Currently I have the `-meta` argument included for all pipeline runs assuming that the user will almost always input multiple sequences. However, it is possible to run the pipeline on one sequence, so I want to add a function that will detect if the input FASTA has one or multiple sequences and adjust this parameter accordingly